src/filters/filter_cue.cpp

```diff
@@ -416,6 +425,18 @@ namespace dmxfish::filters {
         }
 
 
+    }
+
+    void filter_cue::setup_filter(const std::map <std::string, std::string> &configuration,
```

Comment:
do we need still need the configuration here? or it is just easier for all other filter to do the configuration here instead of implementing also pre_setup?

Reply:
We still need this step as there are information required that are not yet available during `pre_setup`.